modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/accept/language/AcceptLanguageImpl.java

```diff
@@ -78,25 +78,23 @@ else if (acceptLanguage.equals("zh-Hant-TW")) {
 				Locale.LanguageRange.parse(acceptLanguage),
 				companyAvailableLocales);
 
-			List<Locale> fixedLocales = new ArrayList<>();
-
-			for (Locale locale : locales) {
-				for (Locale companyAvailableLocale : companyAvailableLocales) {
-					if (LocaleUtil.equals(locale, companyAvailableLocale)) {
-						fixedLocales.add(companyAvailableLocale);
-
-						break;
-					}
-				}
-			}
-
-			if (ListUtil.isEmpty(fixedLocales)) {
+			if (ListUtil.isEmpty(locales)) {
```

Comment:
Shouldn't this if be after the call to `TransformUtil`? The logic is not equivalent to the previous commit. If the transformutil returns null for all elements the exception will not be thrown.

Reply:
It is not equivalent, but it should not fail. Anyway, as we discussed, I am moving it before the empty validation